elliott/elliottlib/metadata.py

```diff
@@ -258,11 +265,15 @@ def default_return():
                 raise IOError(msg)
 
             def latest_build_list(pattern_suffix):
-                # Include * after pattern_suffix to tolerate other release components that might be introduced later.
-                # Also include a .el<version> suffix to match the new build pattern
+                # Include * after pattern_suffix to tolerate other release components that might be introduced later
+                # we do not want "**" in the pattern so check first
+                main_pattern = f"{pattern_prefix}{extra_pattern}{pattern_suffix}"
+                if main_pattern[-1] != "*":
```

Comment:
`not main_pattern.endswith('*')` reads a bit cleaner to me.

Reply:
thanks, updated